config/quickwit.yaml

```diff
@@ -52,13 +53,13 @@ version: 0
 # Path to directory where temporary data (caches, intermediate indexing data structures)
 # is stored. Defaults to `./qwdata`.
 #
-# data_dir: /path/to/data/dir
+data_dir: ${QW_DATA_DIR:-./qwdata}
 #
 # Metastore URI. Defaults to `data_dir/indexes#polling_interval=30s`,
 # which is a file-backed metastore and mostly convenient for testing. A cluster would
 # require a metastore backed by Amzon S3 or PostgreSQL.
 #
-# metastore_uri: s3://your-bucket/indexes
+metastore_uri: ${QW_METASTORE_URI:-./qwdata/indexes#polling_interval=5s}
```

Comment:
can you revert this file? The default logic is acually more refined than `./qwdata/indexes#polling_interval=5s`